### ISSUE TITLE
NV no longer respects the hotkey

### ISSUE THREAD

**Issue by Nemesis041** [Malcolm-Q/LC-LateGameUpgrades#315]
I rebound the NV hotkey from LeftAlt to CapsLock but it still seems to be activating on LeftAlt. CapsLock was working fine before the recent update, but I'm not sure if this is due to the update or other mods causing this issue.

**Comment by WhiteSpike** [comment]
Apparently it works with "<Keyboard>/capsLock". 

Previously it was using a parser which I did not implement to transform the input string to a keyCode which then we check when pressing. To put it back on, I would have to make a parser which turned string into KeyCode and then to the correct string format for the new binding system.

**Comment by Nemesis041** [comment]
That also doesn't seem to work, apparently.

**Comment by SnackSBR** [comment]
When using Input Utils the biding will go to BepInEx\config\controls\com.malco.lethalcompany.moreshipupgrades.IngameKeybinds.json

`{"overrides":[{"action":"WheelbarrowKey","origPath":"<Mouse>/middleButton","path":"<Keyboard>/capsLock"},{"action":"NvgKey","origPath":"<Keyboard>/leftAlt","path":"<Keyboard>/capsLock"}]}` it will be something like this. Tested capslock for the night vision and it's working as intended. If you are using input utils you really should change it from the game interface. If not using input utils the keybind is indeed `<Keyboard>/capsLock`

**Comment by Nemesis041** [comment]
Yep, I did find I had Input Utils installed, and that did the trick. Thank you.
